mon-pix/app/routes/campaigns/join-restricted-campaign.js

```diff
@@ -14,7 +14,7 @@ export default class JoinRestrictedCampaignRoute extends Route.extend(SecuredRou
 
   async beforeModel(transition) {
     this.set('_isReady', false);
-    const campaignCode = transition.to.params.campaign_code;
+    const campaignCode = transition.to.parent.params.campaign_code;
```

Comment:
ça fonctionnait pas ici `const campaignCode = this.paramsFor('campaigns').campaign_code;` ?